hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/ChildFeeContextImpl.java

```diff
@@ -33,6 +34,12 @@ public class ChildFeeContextImpl implements FeeContext {
     private final Authorizer authorizer;
     private final ReadableStoreFactory storeFactory;
     private final Instant consensusNow;
+    // The verifier is non-null only for batch inner transactions.
+    // Since all other child transactions have no signatures to verify, the verifier is no needed.
```

Comment:
Is this always true, that all other child transactions never have signatures to verify? At the very least, I thought we would need to verify signatures for child scheduled transactions

Reply:
All other synthetic child transactions don't have any signatures to verify. For the scheduled transactions we sign the top level ScheduleSign transaction and not the child transaction.